Include/internal/condvar.h

```diff
@@ -22,70 +22,18 @@
 #define PyCOND_T pthread_cond_t
 
 #elif defined(NT_THREADS)
-/*
- * Windows (XP, 2003 server and later, as well as (hopefully) CE) support
- *
- * Emulated condition variables ones that work with XP and later, plus
- * example native support on VISTA and onwards.
- */
+/* Windows support: use native Win7 primitives */
 #define Py_HAVE_CONDVAR
 
 /* include windows if it hasn't been done before */
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 
-/* options */
-/* non-emulated condition variables are provided for those that want
- * to target Windows Vista.  Modify this macro to enable them.
- */
-#ifndef _PY_EMULATED_WIN_CV
-#define _PY_EMULATED_WIN_CV 1  /* use emulated condition variables */
```

Comment:
Isn't removing this a mistake? I think the `SRWLOCK` version was tried in another issue that was abandoned.

Reply:
Hum, this code seems to sensitive and my change may conflict with https://bugs.python.org/issue29871 so I reverted this change and will open a new dedicated PR.